__tests__/app.test.js

```diff
@@ -339,3 +339,45 @@ describe("PATCH api/reviews/:review", () => {
       });
   });
 });
+describe("GET api/reviews/(queries)", () => {
+  test("returns", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction&sort_by=review_id&order=asc")
```

Comment:
Good job on having one test for all the behaviours but here you are not testing for wether we have sorted them properly. In that sense, please break the three behaviours in three different tests that each checks for something specific, and then build a test that checks for all of the queries :)